modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/task/DownloadEpisodeTask.kt

```diff
@@ -141,7 +142,9 @@ class DownloadEpisodeTask @AssistedInject constructor(
             }
 
             LogBuffer.i(LogBuffer.TAG_BACKGROUND_TASKS, "Worker Downloading episode ${episode.title} ${episode.uuid}")
-            setForegroundAsync(createForegroundInfo())
+            if (!Util.isWearOs(context)) {
+                setForegroundAsync(createForegroundInfo())
+            }
```

Comment:
This is throwing an error on the watch because it triest to create a foreground notification, and notifications aren't working currently.

Reply:
It should be resolved by setting up notification channels, haven't tested it though.

Reply:
I bet you're right. I'll enable notifications in a separate PR.